### PR TITLE
Bump required meson version to 0.53.2

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by bluca** [comment]
CI's unhappy:

```
catalog/meson.build:32:6: ERROR: add_install_script args must be strings
```

File: src/test/meson.build [hunk 1]
```diff
@@ -15,7 +15,7 @@ path = run_command(sh, '-c', 'echo "$PATH"').stdout().strip()
 test_env = environment()
 test_env.set('SYSTEMD_KBD_MODEL_MAP', kbd_model_map)
 test_env.set('SYSTEMD_LANGUAGE_FALLBACK_MAP', language_fallback_map)
-test_env.set('PATH', '@0@:@1@'.format(meson.build_root(), path))
+test_env.set('PATH', project_build_root + ':' + path)
```
Comment (poettering) [inline src/test/meson.build:18]: commit msg: system → systemd in one place

**Comment by poettering** [comment]
not a meson guru, but looks all good to me.

**Comment by keszybz** [comment]
I dropped the commit to revert `'sh'` → `sh`. It seems that a version higher than 0.53.2 is required to make that work. Let's see how that goes.

**Review by bluca — APPROVED** [review]
lgtm

**Pull request merged**